packages/entities/entities-redis-configurations/src/components/RedisConfigurationList.cy.ts

```diff
@@ -0,0 +1,253 @@
+import type { KongManagerRedisConfigurationListConfig, KonnectRedisConfigurationListConfig } from 'src/types'
+import RedisConfigurationList from './RedisConfigurationList.vue'
+import { createRouter, createWebHistory } from 'vue-router'
+import { partials, links } from '../../fixtures/mockData'
+import { v4 as uuidv4 } from 'uuid'
+
+const baseConfigKM: KongManagerRedisConfigurationListConfig = {
+  app: 'kongManager',
+  workspace: 'default',
+  apiBaseUrl: '/kong-manager',
+  createRoute: { name: 'redis-configuration-create' },
+  getViewRoute: (id) => ({ name: 'redis-configuration-detail', params: { id } }),
+  getEditRoute: (id) => ({ name: 'redis-configuration-edit', params: { id } }),
+}
+
+const baseConfigKonnect: KonnectRedisConfigurationListConfig = {
+  app: 'konnect',
+  controlPlaneId: 'test-control-plane-id',
+  apiBaseUrl: '/us/kong-api',
+  createRoute: { name: 'redis-configuration-create' },
+  getViewRoute: (id) => ({ name: 'redis-configuration-detail', params: { id } }),
+  getEditRoute: (id) => ({ name: 'redis-configuration-edit', params: { id } }),
+}
+
+describe('<RedisConfigurationList />', () => {
+
+  describe('actions', {
+    viewportHeight: 700,
+    viewportWidth: 700,
+  }, () => {
+    function interceptList({
+      status = 200,
+      body = partials,
+    }: {
+      status?: number,
+      body?: any,
+    } = {}) {
+      cy.intercept({
+        method: 'GET',
+        url: `${baseConfigKM.apiBaseUrl}/${baseConfigKM.workspace}/partials*`,
+      }, {
+        statusCode: status,
+        body,
+      }).as('getRedisConfigurations')
+
+      cy.intercept({
+        method: 'GET',
+        url: `${baseConfigKonnect.apiBaseUrl}/v2/control-planes/${baseConfigKonnect.controlPlaneId}/core-entities/partials*`,
+      }, {
+        statusCode: status,
+        body,
+      }).as('getRedisConfigurations')
```

Comment:
We need to make sure assigning an alias twice would not cause any issues.

Reply:
anti-pattern, fixed